Utils/Elasticsearch/mapping/prodsys.mapping

```diff
@@ -175,24 +175,47 @@
               "ignore_above" : 256
             }
           }
-        },
-        "k_factor": {
-          "type": "keyword",
-          "ignore_above" : 256
-        },
-        "cross_section_ref": {
-          "type": "keyword",
-          "ignore_above" : 256
-        },
-        "cross_section": {
-          "type": "keyword",
-          "ignore_above" : 256
-        },
-        "gen_filt_eff": {
-          "type": "keyword",
-          "ignore_above" : 256
         }
       }
-    }
+    },
+    {
+    "output_datasets": {
+        "_parent": {
+             "type": "task"
+         },
+         "properties": {
+             "datasetname" : {
+               "type" : "text",
+               "fields" : {
+                  "keyword" : {
+                      "type" : "keyword",
+                      "ignore_above" : 256
+                  }
+               }
+             },
+             "bytes" : {
+                 "type" : "long"
+             },
+             "deleted": {
```

Comment:
Why to add Rucio parameters here? They are added in PR #85. And has nothing to do with the current AMI-branch.
I see two variants: 
1) to change mapping as if there were no Rucio parameters to join mappings later during rebase/merge (but please check the mapping from #85 carefully to avoid conflicts: e.g. `output_datasets` are called `output_dataset` there to correspond `task`, which are singular; after all, it is "type" of a document, so why would the "type" name be plural?)
OR
2) to cherry-pick commit 68a8252 from PR #85, and add AMI parameters in the next commit.

I believe that the second variant would be the easiest one -- unless we change something in the mapping before everything is merged. When we rebase this PR on top of the `master` after #85 is merged, this cherry-picked commit will become empty and thus will be removed automatically, while the AMI-related one will stay where it must be. Thus we will escape possible conflicts due to same parts of one file changed twice.

And I think it will be good to rename the file according to the `template` field. When we load the mapping to ES, it is named after the file name, while `template` is used for applying this mapping to the matching indices. Nothing wrong in having this two out of sync, it just makes easier to make a mistake and drop wrong template if needed ;) Or have two mappings for same index -- I don`t know how it`s gonna work, by the way.

Reply:
I did the 2nd variant and think it's must be OK now. Thank you for noticed this mistake :)

Reply:
You`re most welcome. Now check the diff ;)